DeltaEngine/Engine/Input/DeltaInputs.hpp

```diff
@@ -153,6 +153,11 @@ struct Input
 	int wheelVertically;
 };
 
+struct MousePosition{
```

Comment:
We hebben een Point klasse

Reply:
ik zie hem idd dacht dat die alleen maar float was. maar point nu gebruikt